packages/calling-stateful-client/src/Converter.ts

```diff
@@ -27,11 +27,9 @@ import { CaptionsInfo } from './CallClientState';
 import { _isACSCall } from './TypeGuards';
 import { CallCommon, IncomingCallCommon } from './BetaToStableTypes';
 
-/* @conditional-compile-remove(video-background-effects) */ /* @conditional-compile-remove(optimal-video-count) */
+/* @conditional-compile-remove(optimal-video-count) */
```

Comment:
We should be removing this CC line as well (also stabilized but will make this cleaner)